Userland/DevTools/HackStudio/HackStudioWidget.cpp

```diff
@@ -869,7 +869,7 @@ ErrorOr<NonnullRefPtr<GUI::Action>> HackStudioWidget::create_remove_current_edit
 
 ErrorOr<NonnullRefPtr<GUI::Action>> HackStudioWidget::create_toggle_open_file_in_single_click_action()
 {
-    return GUI::Action::create_checkable("&Toggle Open File In Single Click", [this](auto&) {
```

Comment:
Please squash this commit into the previous one

Reply:
Squashed the original 2 commits into 1.